iothub/device/src/IotHubBaseClient.cs

```diff
@@ -208,6 +218,11 @@ public async Task SendTelemetryAsync(TelemetryMessage message, CancellationToken
         /// <exception cref="OperationCanceledException">The operation has been canceled.</exception>
         /// <exception cref="IotHubClientException">An error occured when communicating with IoT hub service.</exception>
         /// <exception cref="ObjectDisposedException">The client has been disposed.</exception>
+        /// <example>
+        /// <code language="csharp">
+        /// await client.SendTelemetryAsync(new List<TelemetryMessage/> { message1, message2 }, cancellationToken);
```

Comment:
forward slash is needed for `List<TelemetryMessage/>` in XML, otherwise it will complain with [CS1570](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs1570).

Reply:
Also, should show initializing the client with AMQP, as it is not the default, with a comment that this only works over AMQP.

Reply:
> I think you need to replace the <> signs with < and >, actually.

Tried with `new List < TelemetryMessage > ` but still complains.

Reply:
Sorry, what I wrote didn't display right. Let me try again with a code block:

```
&lt;TelemetryMessage&gt;
```

Reply:
I could be wrong. Perhaps we reach out to Alma Jenks and ask how to format these correctly in doc comment code blocks?

Reply:
> &lt;TelemetryMessage&gt;

Actually this works!